bciers/libs/shared/components/src/lib/datagrid/DataGrid.tsx

```diff
@@ -51,13 +51,15 @@ const DataGrid: React.FC<Props> = ({
   const { replace } = useRouter();
   const isRowsEmpty = rows.length === 0;
 
+  // brianna I don't think we need this, the second use effect should do it
   useEffect(() => {
     setIsComponentMounted(true);
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   useEffect(() => {
+    // brianna can the check we on if we have props?
```

Comment:
I think I understand what you're getting at here though I may be misinterpreting this grammar haha, though if this was:

` if (!props || !fetchPageData) return;`

then it would never run if there was no initial data?

Reply:
These are leftover; since I'm switching gears I'm not going to change anything about the component. As for what I was thinking here... I think I was confused by the comment on line 64 about server side props vs intialData